static/app/components/smartSearchBar/index.tsx

```diff
@@ -864,6 +867,68 @@ class SmartSearchBar extends Component<Props, State> {
     return this.findTokensAtCursor(matchedTokens);
   }
 
+  /**
+   * Get the active filter
+   */
+  get cursorFilter() {
```

Comment:
Damn did we not already have something like this in this file 🤯

Reply:
We do have `cursorToken`, but that matches `Token.ValueText` as well, which I didn't want